components/NavigationBar/tests/unit/NavigationBarTests.m

```diff
@@ -199,6 +200,32 @@ - (void)testTitleFontProperty {
   XCTAssertEqual(fontTraits, resultTraits);
 }
 
+- (void)testTitleFontPropertyWithAllowAnyTitleFontSizeEnabled {
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+  navBar.frame = CGRectMake(0, 0, 300, 25);
+  navBar.title = @"this is a Title";
+  navBar.titleAlignment = MDCNavigationBarTitleAlignmentCenter;
```

Comment:
The four lines above looks like often-repeated code. Can you move it into a helper method?

Reply:
Removed the excess initializations.